src/main/java/bio/terra/service/resourcemanagement/BufferService.java

```diff
@@ -97,6 +110,17 @@ public ResourceInfo handoutResource() {
           "Retrieved resource from pool {} on Buffer Service instance {}",
           bufferServiceConfiguration.getPoolId(),
           bufferServiceConfiguration.getInstanceUrl());
+
+      if (enableSecureMonitoring) {
```

Comment:
Assuming we're going this here and not `GoogleProjectService` so that we turn this on ASAP?

Reply:
Yup! This way, you cannot get a project from RBS that should be in a secure folder, but isn't.